scripts/cli

```diff
@@ -50,7 +50,13 @@ cmd_build_json2type () {
   set +e
   npm i
   npm run clean
-  npm run build:server
+
+  # With yarn 3, the 'test/' folder is not present, so all built files are put directly in the
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏out of curiosity, how was created this test folder?

Reply:
ok, makes sense!